### PR TITLE
Define styling for default plone search.

### PR DESCRIPTION
**Pull request by bierik**
Depends on https://github.com/4teamwork/ftw.theming/pull/44

Livesearch:
bildschirmfoto 2016-03-07 um 14 59 37
bildschirmfoto 2016-03-07 um 14 59 51

Searchpage:

bildschirmfoto 2016-03-07 um 15 00 00
bildschirmfoto 2016-03-07 um 15 00 10

### CODE REVIEW

**Comment by jone** [comment]
@bierik which ftw.theming PR does this depend on? The nr is missing in the URL :wink:

File: plonetheme/blueberry/scss/site/layout.scss [hunk 1]
```diff
@@ -74,13 +74,25 @@
 }
 
 #portal-logo {
+  @include screen-small() {
+    @include cell();
+    @include gridposition(0);
+    @include gridwidth(8);
```
Comment (jone) [inline plonetheme/blueberry/scss/site/layout.scss]: There's another @include screen-small() with another @include gridwidth(10); below (line 93).
Isn't the one below overwriting this one? Can't we have only one screen-small?

**Comment by jone** [comment]
:+1: looks good!
does this already include solr?

**Comment by bierik** [comment]
@jone Depends on https://github.com/4teamwork/ftw.theming/pull/44

**Comment by bierik** [comment]
@jone The solr integration is coming

**Comment by bierik** [comment]
@jone updated

**Pull request merged**